control-plane/tests/test_cli_inspection.py

```
@@ -1315,6 +1355,346 @@ def test_cli_renders_case_detail_with_evidence_provenance_and_cited_advisory_out
+                base_url = f"http://127.0.0.1:{servers[0].server_port}"
+
+                def post_json(path: str, payload: dict[str, object]) -> dict[str, object]:
+                    response = request.urlopen(
+                        request.Request(
+                            f"{base_url}{path}",
+                            data=json.dumps(payload).encode("utf-8"),
+                            headers={"Content-Type": "application/json"},
+                            method="POST",
+                        ),
+                        timeout=2,
+                    )
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)